fixbackend/subscription/router.py

```diff
@@ -54,13 +50,17 @@ async def aws_marketplace_fulfillment(
     async def aws_marketplace_fulfillment_after_login(
         request: Request,
         maybe_user: OptionalAuthenticatedUser,
+        market_place_handler: AwsMarketplaceHandlerDependency,
         fix_aws_marketplace_token: str = Cookie(None, alias="fix-aws-marketplace-token"),
     ) -> Response:
         if maybe_user is None:  # not logged in
             add_url = request.scope["router"].url_path_for(AddUrlName)
             return RedirectResponse(f"/auth/login?returnUrl={add_url}")
         elif (user := maybe_user) and fix_aws_marketplace_token is not None:  # logged in and token present
-            await market_place_handler().subscribed(user, fix_aws_marketplace_token)
+            subscription = await market_place_handler.subscribed(user, fix_aws_marketplace_token)
+            if subscription.workspace_id is None:  # no workspace yet
```

Comment:
If the user only has one workspace - it should be assigned automatically.

Reply:
Already happening in the handler